music-journal/components/AddMemory.js

```diff
@@ -1,12 +1,23 @@
+import React, { useState } from 'react';
 import { Row, Col, Form, Button } from 'react-bootstrap/';
+import { addMemory } from '../utils/fetcher'
 import withAuth from '../utils/withAuth';
-import submitMemory from '../utils/submitMemory';
+
 const AddMemory = (props) => {
-    const addMemory = () => {
-        submitMemory({
-            song: document.getElementById("song").value,
-            memorytext: document.getElementById("memorytext").value,
-        }, props.token);
+    const [song, setSong] = useState('')
+    const [memoryText, setMemoryText] = useState('')
+
+    const handleSubmit = async (event) => {
+        event.preventDefault()
```

Comment:
Should we be using ; at the ends of lines? I don't know the convention :D

Reply:
TIL about event.preventDefault()! I gess in this context it'll mean that if the event passed into this function has a default behaviour, that will be cancelled?

Reply:
I think we can setup eslint and then I prefer not using ; but it's a style preference really.

So preventDefault is often used to stop the page refreshing on submit.